packages/components/icons/src/custom-icon/custom-icon.styles.ts

```diff
@@ -0,0 +1,27 @@
+// use negative margin to make it so that border doesnt shift content?
```

Comment:
Could you maybe explain this comment? 🙏

Reply:
this was left here accidentally, it was a note to myself that I didnt end up using/following, will delete, thanks for pointing it out!